### PR TITLE
Update lab-07-3-linear_regression_min_max.py

### PR DESCRIPTION
**Pull request by qoocrab**
1. Changed the function name because there are many inquiries that it is confused with 'MinMaxScaler' of 'sklearn.preprocessing'.
2. Added normalized output.

### CODE REVIEW

**Comment by kkweon** [comment]
Thank you!
